text/0000-more-api-documentation-conventions.md

```diff
@@ -0,0 +1,409 @@
+- Feature Name: More API Documentation Conventions
+- Start Date: 2016-03-31
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+[RFC 505] introduced certain conventions around documenting Rust projects. This RFC supersedes
+that one, though it has the same aims: to describe how the Rust project should be documented,
+and provide guidance for other Rust projects as well.
+
+This RFC will contain some similar text as RFC 505, so that we can have one RFC with the full
+conventions.
+
+[RFC 505]: https://github.com/rust-lang/rfcs/blob/master/text/0505-api-comment-conventions.md
+
+# Motivation
+[motivation]: #motivation
+
+Documentation is an extremely important part of any project. It’s important
+that we have consistency in our documentation.
+
+For the most part, the RFC proposes guidelines that are already followed today,
+but it tries to motivate and clarify them.
+
+# Detailed design
+[design]: #detailed-design
+
+This RFC is large. Here’s a table of contents:
+
+* [Content](#content)
+  * [Summary sentence](#summary-sentence)
+  * [English](#english)
+* [Form](#form)
+  * [Use line comments](#use-line-comments)
+  * [Using Markdown](#using-markdown)
+* [Example](#example)
+
+## Content
+[content]: #content
+
+These conventions relate to the contents of the documentation, the words themselves.
+
+### Summary sentence
+[summary-sentence]: #summary-sentence
+
+In API documentation, the first line should be a single-line short sentence
+providing a summary of the code. This line is used as a summary description
+throughout Rustdoc’s output, so it’s a good idea to keep it short.
+
+The summary line should be written in third person singular present indicative
+form. Basically, this means write “Returns” instead of “Return.”
+
+### English
+[english]: #english
+
+This section applies to `rustc` and the standard library.
+
+All documentation is standardized on American English, with regards to
+spelling, grammar, and punctuation conventions. Language changes over time,
+so this doesn’t mean that there is always a correct answer to every grammar
+question, but there is often some kind of formal consensus.
+
+One specific rule that comes up often: when quoting something for emphasis,
```

Comment:
Yes, it is something that's done deliberately, though I am not aware if it's a regionalism or not.

I've often seemed bolding used when showing off a term for the first time, or to highlight things for other reasons. Maybe this is too specific, though. Right now, all the docs use quoting for emphasis, and use bold very sparingly.